code/game/gamemodes/blob/theblob.dm

```diff
@@ -124,24 +128,33 @@
 			else	T = null
 
 	if(!T)	return 0
-	var/obj/effect/blob/B = new /obj/effect/blob/normal(src.loc)
+	//We can keep the above code in this proc, it probably works fine?

```

Comment:
generally we avoid making too many comments like this one and the one below it.

As a rule of thumb, if the comment doesn't aid in understanding the code then don't bother with it.